### PR TITLE
GH-950 Replaces matches with filter in PropEdit

### PR DESCRIPTION
**Pull request by jbrundage**
Fixes GH-950

@GordonSmith @dtsnell4 Please review

Signed-off-by: Jay Brundage <jaman.brundage@lexisnexis.com>

### CODE REVIEW

File: src/other/PropertyEditor.js [hunk 1]
```diff
@@ -92,7 +92,7 @@
     PropertyEditor.prototype.enter = function (domNode, element) {
         Accordion.prototype.enter.apply(this, arguments);
         
-        if(!this._parentElement.node().matches(".other_PropertyEditor div")){
+        if(this._parentElement.filter(".other_PropertyEditor div")[0].length === 0){
```
Comment (GordonSmith) [inline src/other/PropertyEditor.js]: I am not 100% sure what your testing for here, but would the following work:
```
if(this._parentElement.filter(".other_PropertyEditor div").empty()){
```

Comment (jbrundage) [inline src/other/PropertyEditor.js]: Yes, it would. It's a conditional to test whether or not "this" is the outer-most instance of PropertyEditor.

Comment (GordonSmith) [inline src/other/PropertyEditor.js]: I prefer the empty() method to the ```[0].length === 0``` as the in the latter you are breaking the object encapsulation (your assuming knowledge of the inner workings which may change in the future).

Comment (jbrundage) [inline src/other/PropertyEditor.js]: Agreed, ill make the changes now

**Pull request merged**